docs/tutorials/earth_surface_water.ipynb

```diff
@@ -0,0 +1,1339 @@
7cs9k0DhDgNKcBokgAJQGOLGmakucxeS4AVlU1lKWADMJkqNjtBvb7CmMSlsuc2Sw57VPPqet6mkaS/7bt3TOjnVkBbQtlKXIFVVVjTA3UBEHvnsuIupafLOuJIpFiCsOQJEkYBoMYQU5GcFNMMcUUU0zxmwgprHb47nyLMhk0//XyOV6iSYEBLdB7ZvTgcmspLHbd4CQoZf2h/k/a/OMLkmPwQ9cI2m193gmthVH1fpDjy9+9ZIrvXlZpynExW2WRtAAr+fKYIaHF7xA1WJaO75quqx0TtT87L7l+3nEcLYrKmkrAAS/LpA1b6p13XphVRoV1+29RvwsQ9rUxIE1mvpHmXPLHd6nrmk9OsUNZEsJCLd21xYShcaCHl8KS47Zu7QIQA+qXocbcHmDxHfFeenQYmlEBWs/FM9O9v0czGuMxU+JdYIMHX94uvnsw4Hx7UIaBvv/xfTzbyzD+/1ja6fx4fm6+fXwP0vj3Pr6Oceg1nzMvxtv+/ft6+5i67fgc1ZslHOXc2uzonx/9V4v8Mof1/o3nlTYhKmtAf/z5nbOdFFzSdYSaq+t986yIt8fD7fU0N3wDpZ6nykeNr10/bzzIqHNW55p122gdYyzJ5OVoe0X/pphiiimmmOK3FL9WIEI7qX/4w3/KT37yc8pywzAUxHFAVVXM5wltO1DXB5qmdObS/YmlANplJF/kXSed7t44zH8RP6b9SpIwNoYNR0BE4JJzXUDoF7J+eQ+P9u2TC+8BIcBD3zd0Xejeo7JIMNZzleJlx3Z7dDr0ItGSpild13B7O6PrZjSNFG6TpCBJDPN5hiQPkkgkiXTvWBsh/gsxwmjQRL1FNG21C711RU8dyzEQcb7YUA3btg0ZBkvTBJSlSGj1vRRRl8uMJLGumzwmihJms4zlMqfrKrrO0LaaYHV03YFvv/2Gf/Wv/r/80R91vP/+75CmOcMgRfW+h7oOiKIEazPSdCCKepqmpCwrttsFVTUjzztWq8ZJeWniKJ1Zos8rXextywiIkGKxNwjWxZpI+8RxSJomBEFGXQtLoWmkMB5FhihaYMwtxswfzemKsnwgSRaEYToCKB6HAC7SUZdyfX3Nj3/8GQ8P39E0D5TlmijaMwwvuL9f8eWXOR9+uGO5zDkev+L58//I8+e/4LvvXnI8FkRR6zwPROO4aQI2mx1x3PPwsCWKZpRlxcPDa7bbDVVVMwyQJCk3N1f86Ec/4A/+4Md8/PEfEEVXo2T4VwlZnPkOqLcXQTKnaqzdYcyRLCtRdpDvyunc75qEi6HgBx8MrsOtwtqKLEt48eJIVckitus6R9/vHFtG5pB4dOSsVjlXV2Luvd0W7HZHjIGiKOi6luOxo6pqoHL7wUkrBSfAQYwMZcElXi3GeU4EDgwJaJqBtu2pqgZJ6A8EQXtiWvR9QtcldF1AmkbOf0S8IqIoQyXFpphiiimmmGKK30QMLtcHyY8jx5ZUMMAbt2rnM4BKCnmmgMoTtaNicui83UALypJraxFwcGuMDpU5GRsEg+9216KpZ3DLvrRoaYwUOlVyVUEIWRsNrrlh3A3upZEUePBd4ePueO2Ibh1boKDvC8RDrePvMi/WXE6krJQhYZyXgsrsKGtDx9b716mkqXg8qJlvO1r7+fxOxjhm3GEu541bAygQMe7ibk7ggXTB17Tt2HBb/ta2jHLL1snbip6/l1dSPX1OefMw1Pg1lDDgpSFNpV+VEeKLzB7IEt+JMTPnvKg+Bg48MOUL2+4vg4IDoztz1rXPo/2MgYnHYM7YN2EMTrwNkPl9+rW3vmdcHPeGzv7adH7r/8/Pd2yaHY6OPT7/c7BiDNL5f/175HzU3Bz8/Roc4CDseM+ACUfHGv+cn6tnQOjcV/BRx3cYnZM3bVcmhh8Pfz1vS2wFp2P53/3zdQ7O+XqGSlPLtfu57j9r1GRdf2RMBIzzElUTEDHFFFNMMcVvO37NQIRE28oXXpaFXF6mBMGSum6IosjJnDREUU3btiOAQZNoc0qIfWeCaIN6k7dxpw5woou+68cn42pIBwpGjE2szw2f5F+vk6rAhBhFNS456E7Jiu+MEbpw24aUpXQalGV9AkW0kJplCVEUI/4XA1XVOF8GT7MMgsF1WGuXlCbCEQJIyDWpxqV0A4kXhTGScEF82p+wIJRSbAjDEGtbkqQmTQ+kaeS6kyxxHBDHWqSPyLKMxWLOarViuVyRJAfu7go2myWHw0dcXCzpuoE4zvnpT39BUfR8+mnJs2fvs1hAksQEQe1+SoIgQ3wbrjEmIE2h60LquscYiKIUTx3HjYfIFQkFO3DGwa0rZlvCMCUMB6IoJo5z0nRGHM+Yzeak6YwgwLEUOg6HNxyPD8TxQBQlDgw5UFUJUTRDHw1dcAxDgyys3n5ktDPO2uT0WhiGvPfeR/zJn7xgv3/gxYt7qmrg4WFDXYv599/+7b/nk08+48/+7M/59//+Bd99dwQCBzwFtG1LXffOvK/nxYt7/vzPf8GXX74BYpqmoywLiqKkrmuGYSBJElarLa9e7Xnx4g1/+Iff8vnnv8v19YfE8TUCLOhc/T7xrsXp+HeDsnQ8MJiPthmQRWKD10+Vxd1qFZKmGbNZzpMnV1xefkuWveDhYUdVVRwONYdDCwROAkk0mK21Trqpc5JKYgwPOXEsMlvHY8Ph0HA4iLxSmhry3BDHOUkiwEHTxHSddMa1be+eb1loGxOewAr5HAjcNr373GppW28OLwteWRCIsXvIxcUlWbYiSQzWTkn+FFNMMcUUU/wm4rxhYnDFyPNiphabtTnCF/4ea6ifMycB1DNCzJ3V1FmbGDRHB9+hHY4KvAPjNc25vI2+UcCSvpc1inhLBU7+ST0kZB+SczZubSSeaP5fDz54Hwq95t4BAOox17ji5GmERoVgLbB2jj1QuaYN68AIvVbdVrvBxdBbQIOeKOqJIgiCiGGIEKZK4PbbnQAfAY4ESFC5TTlvvWfaWKZAD6iclRbIvbH4/4
```

Comment:
Can we use `kornia.augmentation.Normalize` instead? Looks identical

Reply:
Indeed... just had to "workaround" to avoid normalizing the spectral indices... 
```
# normalize = MyNormalize(*calc_statistics(train_imgs))
mean, std = calc_statistics(train_imgs)

# Please, note that we will create spectral indices using the raw (non-normalized) data. Then, when normalizing, the sensors will have more channels (the indices) that should not be normalized.
# To solve this, we will add the indices to the 0's to the mean vector and 1's to the std vectors

mean = np.concat([mean, [0, 0, 0]])
std = np.concat([std, [1, 1, 1]])

norm = K.Normalize(mean=mean, std=std)

tfms = torch.nn.Sequential(
    indices.AppendNDWI(index_green=1, index_nir=3),
    indices.AppendNDWI(index_green=1, index_nir=5),
    indices.AppendNDVI(index_nir=3, index_red=2),
    norm,
)
```